stuffs/litegapps.py

```diff
@@ -0,0 +1,49 @@
+import os
+import shutil
+from stuffs.general import General
+from tools.helper import get_download_dir, host, print_color, run, bcolors
+
+class LiteGapps(General):
+    # https://master.dl.sourceforge.net/project/litegapps/litegapps/x86_64/33/lite/2024-02-24/AUTO-LiteGapps-x86_64-13.0-20240224-official.zip?viasf=1
+    dl_link_fmt = "https://master.dl.sourceforge.net/project/litegapps/litegapps/{0}/{1}/lite/2024-02-24/AUTO-LiteGapps-{0}-{2}-20240224-official.zip"
```

Comment:
Thanks for providing an alternative to OpenGapps!

Not all links follow this format, it took me a while to get them
https://github.com/ayasa520/redroid-script/blob/main/stuff/litegapps.py#L8-#L177

Reply:
you're the best!